### ISSUE TITLE
Snes9x Next broken when concurrently loading ROMs [Wii port]

### ISSUE THREAD

**Issue by roflcopter777** [libretro/RetroArch#3254]
RetroArch version 1.3.6 Wii
Homebrew Channel: 1.1.2
SD card: 16 GB Sandisk SDHC, FAT32
Console used; Wii U's Wii/vWii Mode, 5.3.2 firmware

Issue: Upon initially loading up RetroArch and going to the Snes9x Next core, any ROM will load up fine, save states, load states work fine as well. However, whenever you concurrently load another ROM after the first one, the emulator hangs and is stuck on a black screen with the text "RESOLUTION" on the bottom-left corner. 

Video of error in action: https://www.youtube.com/watch?v=HDWHhv_7icI

This has happened to me twice, although the last time it happened, it occurred on the first ROM load after changing from the regular Snes9x core to Snes9x Next, so it may be random, I don't know. All I know is that last version of Snes9x Next, despite having completely broken save/load state support that results in an immediate code dump and lockup of the emulator and console.  I don't know if this is enough evidence, or even worth your guys' time, given how efficacious my bug reports have been in the past. 

To recap: 
- All previous RA config files were deleted before loading it up for the first time
- Only setting I changed was the default content browser directory (as it never saves where I loaded the previous ROM, forcing me to navigate all over again to the folder), and the aspect ratio, the only two settings changed. 
- Using the normal Snes9x core works fine, switching to the Next core and loading game after game results in immediate crash of the console, forcing a hard reboot.

**Comment by fr500** [comment]
What if you close content and then load again?

**Comment by roflcopter777** [comment]
I haven't thought of that, I'll try it and report back, I usually don't do that, but I'll give it a shot. Oddly enough, the regular Snes9x (non Next) core doesn't exhibit this behavior, nor does the Genesis Plus core,  I'll try to isolate it further.

**Comment by roflcopter777** [comment]
@fr500  It uh, how do I put it, well I loaded the Snes9x Next core, loaded just one ROM and it locked up, completely, I didn't have a chance to even show the title screen on the game in question. The word "RESOLUTION was plastered on the bottom-left corner, maybe it's because I'm loading it on the Wii U's Wii mode and not a real Wii? WTF?

**Comment by twinaphex** [comment]
Try this again with the latest sources. It should now properly unload the core after selecting Close Content.

**Comment by roflcopter777** [comment]
Well, I hope that I'm able to load a game as last time loading one ROM caused the crash, will report back.
